Projects/views.py

```diff
@@ -95,7 +95,7 @@ def home(request):
         except EmptyPage:
             issues = paginator.get_page(paginator.num_pages)
             issues = Issues.objects.none()
-        return render(request, 'Projects/home.html', {'issues': issues, 'val':val}) #this dic will have value of
```

Comment:
why did you remove this val var?

Reply:
Oh, I am sorry sir recently I was doing some debugging so for that I removed that value and forgot to put that back :sweat_smile: